frontend/packages/db-structure/package.json

```diff
@@ -30,12 +32,12 @@
     "cp:prism": "cp node_modules/@ruby/prism/src/prism.wasm dist/parser/schemarb/",
     "fmt": "pnpm run '/^fmt:.*/'",
     "fmt:biome": "biome check --write --unsafe .",
+    "gen": "pnpm run '/^gen:.*/'",
+    "gen:cp-prism": "cp node_modules/@ruby/prism/src/prism.wasm src/parser/schemarb/prism.wasm",
+    "gen:gen-tbls-schema": "node scripts/tblsJsonSchemaToZod.mjs",
     "lint": "pnpm run '/^lint:.*/'",
     "lint:biome": "biome check .",
     "lint:tsc": "tsc --noEmit",
-    "postinstall": "pnpm run '/^postinstall:.*/'",
-    "postinstall:cp-prism": "cp node_modules/@ruby/prism/src/prism.wasm src/parser/schemarb/prism.wasm",
-    "postinstall:gen-tbls-schema": "node scripts/tblsJsonSchemaToZod.mjs",
```

Comment:
tested. good!

- ✔️  `GITHUB_TOKEN=$(gh auth token) pnpm changeset version` 
- ✔️  `git clean -fdx && asdf local nodejs 22.11.0 && pnpm i && pnpm dev`
      - (`asdf local nodejs 22.11.0` is specific to my local setup 🙏 )
- ✔️ `docker run --rm -it node:20-bullseye npx @liam-hq/cli@0.0.0-next-20250205094432 --version` worked